README.md

```diff
@@ -0,0 +1,48 @@
+# Building container images with Ansible
```

Comment:
Would be great to have lint to ansible-bender with small about line to get quick context when I land on this page.